### ISSUE TITLE
Automatically adjust for time zone display during travel

### ISSUE THREAD

**Issue by poldim** [babybuddy/babybuddy#611]
We just travel to see my in-laws a few time zones away. It would be great to designate a home time zone and have it auto adjust while traveling.

**Comment by cdubz** [comment]
There have been feature requests in the past for using the browser reported timeline but I’m not sure if that’s the same thing. Anyway it would be nice to have more automated handling like this.

**Comment by Birdheezy** [comment]
We are traveling 1 time zone away and while the "time since" is correct the activity start/stop are 1 hour off. 
So "last feeding" shows 1hr and 28min ago which is correct, but just started at nap timer at 9:25 local and it shows 8:25 on the timeline. Would be a nice fix. 

I guess a workaround would be to change the time zone of the docker container?

**Comment by wpyoga** [comment]
I believe the good UX would be to have the time zone be set from the browser. Database entries can be UTC instead of following the system (or docker container) time zone.

Think about it this way: the relevant caregiver is usually (always?) with the baby. So when traveling, the timezone from the caregiver's point of view would be the one that "makes sense".

What about events in a previous timezone? One option is to show those as "time - timezone", so maybe something like "1:45 p.m. - EST". Another, simpler option, would be to show all times in the current browser time zone.

If time zone is set in the server (backend), the good UX would be to set the time zone for the child, for the same reason: the caregiver is usually with the baby. If there are 2 caregivers (mom and dad for example), the one traveling with the child would be the one updating the events.

**Comment by cdubz** [comment]
@wpyoga yes I agree if we could derive and use it from the browser it would be best. I _think_ the existing database handling all happens in UTC so theoretically this is possible as a non-breaking enhancement.
